jwst/datamodels/schemas/core.schema.yaml

```diff
@@ -1228,6 +1228,14 @@ properties:
                 title: Mask reference file name
                 type: string
                 fits_keyword: R_MASK
+          pathloss:
+            title: Pathloss reference file information
+            type: object
+            properties:
+              name:
+                title: Pathloss reference file name
+                type: string
+                fits_keyword: R_PTHLOS
```

Comment:
I guess we should run the spelling of this keyword past Todd for CRDS compatibility. CRDS usually assumes that it can just take the first 6 chars of the ref_type, which in this case is "PATHLO" and stick an "R_" on the front of it in order to figure out which keyword to populate with the name of the ref file used.